enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -588,13 +588,12 @@ func (c *ociContainer) createSpec(cmd *repb.Command) (*specs.Spec, error) {
 				Source:      filepath.Join(c.bundlePath(), "hosts"),
 				Options:     []string{"bind", "rprivate"},
 			},
-			// TODO: shm
-			// {
-			// 		Destination: "/dev/shm",
-			// 		Type:        "bind",
-			// 		Source:      "/run/containers/storage/overlay-containers/99133d16f4f9d0678f87972c01209e308ebafc074f333822805a633620f12507/userdata/shm",
-			// 		Options:     []string{"bind", "rprivate", "nosuid", "noexec", "nodev"},
-			// },
+			{
+				Destination: "/dev/shm",
+				Type:        "tmpfs",
+				Source:      "tmpfs",
+				Options:     []string{"rw", "nosuid", "nodev", "noexec", "relatime", "size=64000k", "inode64"},
```

Comment:
Not sure which of these isn't supported in CI. Could we maybe print the output of something like `journalctl` to see kernel logs when the test fails?

Reply:
The failures are happening only in dev and seem to be fixed after this revert PR https://github.com/buildbuddy-io/buildbuddy/pull/7096

Reply:
Does it make sense to make the size configurable via a flag, or do you think we're unlikely to have to change it for running on different machines?